odh/overlays/moc/monitoring/dashboards/thoth-station/thoth-kafka-argo-metrics.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDashboard
+metadata:
+  name: thoth-kafka-argo-metrics
+  labels:
+    app: grafana
+spec:
+  name: thoth-kafka-argo-metrics
+  url: https://raw.githubusercontent.com/thoth-station/thoth-application/master/grafana-dashboard/overlays/cnv-prod/thoth-kafka-argo-metrics.json
```

Comment:
@harshad16 None of the urls seem to be working for me. I get a 404 error when I try to access them

Reply:
yes, as described on the pull request description they depend upon another PR on thoth-application repository.
once that is merged , these URL will be available and i will unhold this pull request .